python/tank/platform/validation.py

```diff
@@ -620,19 +620,19 @@ def __validate_settings_template(self, settings_key, schema, template_name):
                             (self._context, list(missing_fields), cur_template)
                         )
 
-    def __validate_settings_hook(self, settings_key, schema, hook_name):
+    def __validate_settings_hook(self, settings_key, schema, hook_value):
         """
         Validate that the value for a setting of type hook corresponds to a file in the hooks
         directory.
         """
 
-        if constants.TANK_HOOK_ENGINE_REFERENCE_TOKEN in hook_name:
+        if constants.TANK_HOOK_ENGINE_REFERENCE_TOKEN in hook_value:
```

Comment:
this code was written without debug logging because we didn't have a good logging facility in core back then. Now that we are updating it, could we add some debug logging at the same time?